### PR TITLE
liberate formula

### PR DESCRIPTION
**Pull request by keichwa**
- https://github.com/SUSE/spacewalk/issues/23012
- https://github.com/SUSE/spacewalk/issues/23414

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]

File: modules/specialized-guides/pages/salt/salt-formula-liberate.adoc [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+Base Channel::
+Select one base channel:
++
+
+* EL7: `RHEL x86_64 Server 7`
+* EL8: `RHEL8-Pool for x86_64`
+* EL9: `EL9-Pool for x86_64`
+
+Child Channel::
+Use [literal]``include recommended`` where available or select all if unavailable
```
Comment (mackdk) [inline modules/specialized-guides/pages/salt/salt-formula-liberate.adoc:84]: It turned out that the suggestion to include recommended is not quite correct as discussed in this Slack thread https://suse.slack.com/archives/C02D78LLS04/p1707753031169719.

I think the safest approach is to always include all channels, unless the customer knows exactly what channels are needed.

@rjmateus What do you think?

**Review by mackdk — COMMENTED** [review]
Looks very good to me, there is only one issue we need to address that was discovered during the testing of the feature.

File: modules/specialized-guides/pages/salt/salt-formula-liberate.adoc (hunk 1 above)
Comment (rjmateus) [reply modules/specialized-guides/pages/salt/salt-formula-liberate.adoc:84]: I agree with Thomas, we should say to add all channels. Another option could be say that we need client tools channels and update channels, but this option can get outdated faster.

Comment (keichwa) [reply modules/specialized-guides/pages/salt/salt-formula-liberate.adoc]: thanks for feedback, I changed it accordingly.

**Review by mackdk — APPROVED** [review]

**Pull request merged**